components/Header.tsx

```
@@ -142,53 +188,112 @@ export function Header() {
+                                        aria-hidden="true"
+                                        viewBox="0 0 20 20"
+                                        fill="none"
+                                        className="pointer-events-none absolute left-3 top-1/2 h-4 w-4 -translate-y-1/2 text-[var(--color-muted-foreground)] md:left-4 md:h-5 md:w-5"
                                     >
-                                        {group.label}
+                                        <circle cx="9" cy="9" r="6" stroke="currentColor" strokeWidth="1.5" />
+                                        <path d="M14 14l3 3" stroke="currentColor" strokeWidth="1.5" strokeLinecap="round" />
+                                    </svg>
+                                    <input
+                                        type="search"
+                                        role="searchbox"
```

Biome (2.4.15)

[error] Using the role attribute 'searchbox' on the 'input' element is redundant, because it is implied by its semantic.

(lint/a11y/noRedundantRoles)